backend/websocket.js

```
@@ -0,0 +1,51 @@
+    ws.on('close', () => {
+      clients.delete(ws);
+      console.log('🔌 Client disconnected');
+    });
+
+    // Send initial test message
+    ws.send(JSON.stringify({ type: 'INIT', data: { message: 'Connected to Sentinel WebSocket' } }));
+  });
+  
+  // Ping interval to keep connections alive and detect stale ones
+  const interval = setInterval(() => {
+    wss.clients.forEach((ws) => {
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)